### PR TITLE
fix: LEAP-930: Changes to ML settings UI

### PR DESCRIPTION
**Pull request by hlomzik**
Annotation settings:
- remove links to docs as they don't fully describe the flow

Model settings:
- also remove links
- fix text describing how to use new model in labeling
- add link to Annotation settings
- simplify blocks used on a page

Screenshot 2024-04-10 at 23 58 00
Screenshot 2024-04-10 at 23 57 48

Mirroring #5696 for Follow Merge

### PR fulfills these requirements
- [ ] Tests for the changes have been added/updated (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [x] Best efforts were made to ensure docs/code are concise and coherent (checked for spelling/grammatical errors, commented out code, debug logs etc.)
- [x] Self-reviewed and ran all changes on a local instance (for bug fixes/features)

#### Change has impacts in these area(s)
- [ ] Product design
- [ ] Backend (Database)
- [ ] Backend (API)
- [x] Frontend

### CODE REVIEW

File: web/apps/labelstudio/src/pages/Settings/MachineLearningSettings/MachineLearningSettings.js [hunk 1]
```diff
@@ -132,33 +132,24 @@ export const MachineLearningSettings = () => {
...
-          </Description>
+          <>
+            <Description>
+              A connected model has been detected! If you wish to fetch
+              predictions from this model, please follow these steps:
+              <br />
+              <br />
+              1. Navigate to the <i>Data Manager</i>.<br />
+              2. Select the desired tasks.<br />
+              3. Click on <i>Retrieve predictions</i> from the <i>Actions</i> menu.
```
Comment (yyassi-heartex) [inline web/apps/labelstudio/src/pages/Settings/MachineLearningSettings/MachineLearningSettings.js]: I see this is copy/paste from before but why are we relying on `<br />` for spacing and layout? also this seems like an optimal spot to use `<ol>` for the list - can we get a follow up ticket to address this?

**Review by yyassi-heartex — APPROVED** [review]
approving with the expectation that there will be a follow up ticket to address my concern regarding the layout/component structure of the description content

**Comment by hlomzik** [comment]
/git merge
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/8741386553)
> Successfully merged:  441 files changed, 2628 insertions(+), 2411 deletions(-)

File: web/apps/labelstudio/src/pages/Settings/MachineLearningSettings/MachineLearningSettings.js (hunk 1 above)
Comment (hlomzik) [reply web/apps/labelstudio/src/pages/Settings/MachineLearningSettings/MachineLearningSettings.js]: I'm going to fix this asap, yes, thank you!

**Pull request merged**